src/containerapp/azext_containerapp/commands.py

```diff
@@ -205,6 +205,18 @@ def load_command_table(self, args):
         g.custom_show_command('show', 'show_eureka_server_for_spring')
         g.custom_command('delete', 'delete_eureka_server_for_spring', confirmation=True, supports_no_wait=True)
 
+    with self.command_group('containerapp env java-component nacos') as g:
+        g.custom_command('create', 'create_nacos', supports_no_wait=True)
+        g.custom_command('update', 'update_nacos', supports_no_wait=True)
+        g.custom_show_command('show', 'show_nacos')
+        g.custom_command('delete', 'delete_nacos', confirmation=True, supports_no_wait=True)
+
+    with self.command_group('containerapp env java-component admin-for-spring') as g:
```

Comment:
add is_preview for new command group

Reply:
Line 179 added is_preview for containerap env java-component command group.